CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-.. This file should contain the changes for the last release only, which
+[6~].. This file should contain the changes for the last release only, which
```

Comment:
I usually do, but this one was literally 20 seconds before I had to left :yum: . Thanks for the review.